### ISSUE TITLE
Issue when update to api 25.3.1

### ISSUE THREAD

**Issue by xamarindevelopervietnam** [xabre/xamarin-forms-tab-badge#15]
can't find TabLayout.SlidingTabStrip class when update api to 25.3.1

**Comment by xamarindevelopervietnam** [comment]
line 23 of https://github.com/xabre/xamarin-forms-tab-badge/blob/master/Source/Plugin.Badge.Droid/BadgedTabbedPageRenderer.cs

**Comment by benevbright** [comment]
same here

**Comment by Nerololo** [comment]
Disable linking an will work, i cant get work skipping that dll

**Comment by Nerololo** [comment]
Same here!. Its look like in Android 25.3.1 TabLayout.SlidingTabStrip not exist any more

**Comment by emeric0101** [comment]
To fix it : just change SlidingTabStrip with LinearLayout

**Comment by lanarchyste** [comment]
Thank you emeric0101 !

I will now be able to update some components that required the latest versions of the Android packages. :)

**Comment by benevbright** [comment]
Will you release on the nuget gallery soon?
I'm waiting :)

**Comment by xabre** [comment]
Thanks for your help guys. I've released an updated nuget package with this fix. :)

**Comment by benevbright** [comment]
Hi, I updated to 1.0.2 and I'm getting 
no non-static method "Landroid/widget/TextViewl.setForegroundGravity(I)V"

and here is stacktrace.
![screen shot 2017-06-28 at 3 29 34 pm](https://user-images.githubusercontent.com/916690/27623275-a948b8e0-5c16-11e7-9ef6-35cb3b2ec7e4.png)

Is anyone facing same problem with me?

**Comment by xabre** [comment]
Are you using linking? It seems that the method is removed?

Try adding this to your LinkerPleaseInclude in the android project:

```
void SetForegroundGravity(TextView textView){
    textView.SetForegroundGravity(GravityFlags.Center);
}
```

PS: I added this call in the latest release, I think it's not really needed so I am going to remove it in the next one.

**Comment by benevbright** [comment]
Opps strange, 
I thought I wrote a comment but it is not here.

I tried your help but it does not work. 
And changing Linker setting gives the same result. 

It seems that the method is removed as you said.

**Comment by benevbright** [comment]
It would be great if you could release an update. 
Thanks xabre.

**Comment by xabre** [comment]
Should be fixed with the latest release 1.1.0.
